examples/2d/mesh2d_vertex_color_texture.rs

```diff
@@ -0,0 +1,39 @@
+//! Shows how to render a polygonal [`Mesh`], generated from a [`Quad`] primitive, in a 2D scene.
```

Comment:
This should mention the colored vertexes presumably.

Reply:
I've added a second line (the first is copied from the `mesh2d` example). I wasn't sure how to reference the other example, if that's preferred.